### ISSUE TITLE
Problems building with Java 11

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#1137]
```
ERROR:The instance method compare(CharSequence, CharSequence) cannot override the static method compare(CharSequence, CharSequence) of type CharSequence. (file:/data/xtext-master/git/xtext-core/org.eclipse.xtext/src/org/eclipse/xtext/generator/trace/node/GeneratorNodeProcessor.xtend line : 30 column : 26)
```
seems there is a very ugly api design in Java 11
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/CharSequence.html#compare(java.lang.CharSequence,java.lang.CharSequence)

**Comment by cdietrich** [comment]
@szarnekow is that maybe a Xtend Bug in Override Helper etc?

**Comment by cdietrich** [comment]
minimal reproducing sample

```
class Blala implements CharSequence {
	@Delegate
	CharSequence viech
}
```

**Comment by cdietrich** [comment]
```
public interface IBad {
	
	 public static int compare(CharSequence cs1, CharSequence cs2) {
		 return 1;
	 }
	 
	 public default int demo() {
		 return 2;
	 }

}

```

```
class Blala implements IBad {
	@Delegate
	IBad viech
}
```

**Comment by cdietrich** [comment]
=> looks like an error in the `Delegate` AA

**Comment by cdietrich** [comment]
Fixed in 2.18
